src/webui/api/freediskspacechecker.cpp

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
@glassez 
Should this file placed under src/base (or src/base/utils) or it is fine in src/webui/api?

Reply:
Let's keep it here since it isn't used outside of WebAPI. Or maybe move to `src/webui/api/private` to distinguish from api controllers. I don't think it will be needed somewhere else.